01_install_requirements.sh

```diff
@@ -8,7 +8,7 @@ source sanitychecks.sh
 source utils.sh
 source validation.sh
 
-early_deploy_validation
+early_either_validation
```

Comment:
We still want to exit early if the secret and token aren't set. Maybe we can change the validation step to only run if oc is installed? Or pass an argument to the function to disable that 1 check?

Reply:
Added an argument to the check. PTAL.

Reply:
This script doesn't depend on those variables, so I was thinking it'd be sufficient to leave that check until the 02 script?

I guess alternatively if we want to check those variables are set always, we could move the checks (but not the oc login) to early_either_validation?

Reply:
> This script doesn't depend on those variables, so I was thinking it'd be sufficient to leave that check until the 02 script?
> 
> I guess alternatively if we want to check those variables are set always, we could move the checks (but not the oc login) to early_either_validation?

The "either" function is meant to work for cleanup, too, and the variables aren't needed then.

Even though the requirements script doesn't need the variables if someone types `make` on a clean host I wanted them to not have to wait while it updates all of the packages before they see the error.